files/routes_stats_ft.md

```diff
@@ -0,0 +1,49 @@
+## Additional Transit Route Information
+
+ *  Filename MUST be `routes_stats_ft.txt`
+ *  File MUST contain a record for every transit route (i.e. the Muni 14 Local) 
+ *  File MUST be a valid CSV file.
+ *  The first line of each file MUST contain case-sensitive field names.
+ *  Field names MUST NOT contain tabs, carriage returns or new lines.
+
+File MUST contain ONE the following TWO sets of required attributes:
+
+**Option One**:
+
+Required Attributes	| Description										
+----------			| -------------		
+`route_id`			| ID that uniquely identifies a route.
+`start_date`		| Start date in YYYYMMDD for which route statistics are calculated.
+`end_date`			| End date in YYYYMMDD for which route statistics are calculated.
```

Comment:
do we also need a `service_id`? It is included in https://github.com/ODOT-PTS/GTFS-ride/blob/master/spec/en/reference.md#ridershiptxt